cylc/flow/scripts/stop.py

```diff
@@ -240,10 +240,8 @@ async def _run(
             options.max_polls,
         )
 
-    # mode defaults to 'Clean'
-    mode = None
     if stop_task or stop_cycle:
-        pass
+        mode: Optional[str] = None
```

Comment:
You cannot pass `None` in as a the value for `stopMode` variable... 
The new `graphql-core` expects either a valid argument or no argument at all.

I'll fix this post merge.. somehow..

Reply:
Somehow it works with this command.. and in `graphiql`:
![image](https://github.com/user-attachments/assets/8d66a86a-21c4-497a-acd7-a481fc5a12af)

Not quite sure how, I'm pretty sure it was graphql validation that tripped up complaining that `NoneType` is incompatible with the arg type..

Reply:
Oh, haha, I already commented on it .. 
https://github.com/cylc/cylc-flow/pull/6478#issuecomment-2475788563
it actually passes `None`/`null` through, where before it would use the `default_value` if `null` .. So as long as the resolvers can handle `NoneType` then it's fine..